app/src/main/java/com/hedvig/app/feature/settings/Market.kt

```diff
@@ -82,7 +83,11 @@ enum class Market {
             context.startActivity(ChoosePlanActivity.newInstance(context))
         }
         DK -> {
-            context.startActivity(WebOnboardingActivity.newInstance(context))
+            if (isDebug()) { // TODO NOT COMMIT THIS use for testing the branch
+                context.startActivity(ChoosePlanActivity.newInstance(context))
+            } else {
+                context.startActivity(WebOnboardingActivity.newInstance(context))
+            }
```

Comment:
This was a fast way I did to be able to test this on device, I guess we can make this a feature flag instead.

Reply:
Remember to remove this before we release :)

Reply:
Is this OK https://github.com/HedvigInsurance/android/pull/888/commits/0b017c6f03ec6aa8e95a683a954b1fe5acf0cfd0 for now?
Until when the QA is done